docs/InstallGuide.md

```diff
@@ -131,294 +138,221 @@ To automatically launch the mysql server, execute the following command.
-     cookie=0xc00001100000008, duration=271.027s, table=65, n_packets=0, n_bytes=0, idle_age=271, priority=30,arp,metadata=0x700000000000008/0xff000000ffffffff,arp_op=1 actions=CONTROLLER:65535
-     cookie=0xc00001100000005, duration=272.126s, table=65, n_packets=0, n_bytes=0, idle_age=272, priority=30,arp,metadata=0x700000000000005/0xff000000ffffffff,arp_op=1 actions=CONTROLLER:65535
+The log files are created in the /var/log/openvnet directory. See them if something bad happens. If the vna is successfully launched you can see `is_connected: true` by `ovs-vsctl show` such as following.
 
+```
+fbe23184-7f14-46cb-857b-3abf6153a6d6
+    Bridge "br0"
+        Controller "tcp:127.0.0.1:6633"
+            is_connected: true
+```
 
-Check Virtual Network Operation.
-------------------------------------
+This means the OpenFlow controller which is vna is now connected to the datapath. After the connection between the OpenFlow controller and the
+datapath is established it starts installing the flows on the datapath.
 
-### Start VM(s)
+Verification
+-------
 
-We use kvm to start four virtual machines. Their interfaces were registered in the database earlier. These VMs and their networks are arranged as follows.
+By following all the instructions from the beginning to this section, you already have the sample environment. In this section, we are firstly going to play around the environment then secondary see network packets going through OpenFlow rules to reach
+from one guest to another.
 
-+ vm1, vm3 -> vnet1
-+ vm2, vm4 -> vnet2
+As a representation of the guest here we use [LXC](https://linuxcontainers.org), which helps users run multiple isolated Linux system containers. Any other virtualization technologies might be used, however we take LXC since it is easy to create/destroy/configure and simple enough to do this verification.
 
-The command "run-vm-all" starts all four virtual machines at once.
+Install LXC
 
-    # cd /root/vnet-test-kvm/vm
-    # ./run-vm-all
-    *** vm1 ***
-    node-id: 1
-    interface name: if-x5vjmksl
-    mac address: 52:54:00:0d:84:8c
-    *** vm2 ***
-    node-id: 2
-    interface name: if-sve1s6xp
-    mac address: 52:54:00:9a:cc:00
-    *** vm3 ***
-    node-id: 3
-    interface name: if-bvixz40n
-    mac address: 52:54:00:0a:fb:78
-    *** vm4 ***
-    node-id: 4
-    interface name: if-7st2alal
-    mac address: 52:54:00:e5:f9:fb
+```
+# yum -y install lxc lxc-templates
+```
 
-### Check the IP address of virtual machines
+Create and mount cgroup
 
-You can log into the virtual machines using the telnet command.
+```
+# mkdir /cgroup
+# echo "cgroup /cgroup cgroup defaults 0 0" >> /etc/fstab
+# mount /cgroup
+```
 
-    # telnet localhost PORT-NO
+Create 2 LXC guests
 
-The port numbers are:
+```
+# lxc-create -t centos -n inst1
+# lxc-create -t centos -n inst2
```

Comment:
Creating inst1 worked but inst2 gave me an error because rsync wasn't installed. There should be a warning here that rsync needs to be installed first.